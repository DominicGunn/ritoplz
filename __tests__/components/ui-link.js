'use strict'

/* eslint-env jest */

import renderer from 'react-test-renderer'

import UiLink from '../../components/ui/ui-link'

describe('ui ui-link component', () => {
  describe('snapshots', () => {
    it('should render ui-link', () => {
      const tree = renderer.create(<UiLink href="/">Link</UiLink>).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
