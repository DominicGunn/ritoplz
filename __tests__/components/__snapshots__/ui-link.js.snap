// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui ui-link component snapshots should render ui-link 1`] = `
<a
  className="primary"
  data-jsx={1642471407}
  href="/"
  onClick={[Function]}
  style={undefined}
>
  Link
</a>
`;
